debian/mariadb-server-core-10.1.files

```diff
@@ -1,4 +1,7 @@
 usr/sbin/mysqld
+usr/bin/mariadb-socket-convert
+usr/bin/mariadb-system-convert
+usr/bin/mariadb-systemd-start
 usr/share/man/man8/mysqld.8
 usr/share/mysql/charsets
 usr/share/mysql/czech
```

Comment:
moved to mariadb-server rather than mariadb-server-core - f49e0e9

mariadb-systemd-start removed.